### PR TITLE
Content Review: Salt Guide 1/2

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/8365

### CODE REVIEW

**Comment by Loquacity** [comment]
This marks something pretty close to the halfway point, so I'll start a new PR on Monday.

File: modules/salt/pages/salt-states.adoc [hunk 1]
```diff
@@ -1,97 +1,93 @@
...
-The {productname} salt-master reads its state data from three file root locations.
+[path]``/usr/share/susemanager/salt``::
+This directory is shipped and updated with {productname} and includes certificate setup and common state logic to be applied to packages and channels.
 
-The directory [path]``/usr/share/susemanager/salt``
-It is shipped and updated together with {productname} and includes certificate setup and common state logic to be applied to packages and channels.
-
-The directory [path]``/srv/susemanager/salt`` is generated by {productname} and based on assigned channels and packages for clients, groups and organizations.
+[path]``/srv/susemanager/salt``::
+This directory is generated by {productname} and based on assigned channels and packages for clients, groups and organizations.
```
Comment (keichwa) [inline modules/salt/pages/salt-states.adoc]:
```suggestion
This directory is generated by {productname} and based on assigned channels and packages for clients, groups, and organizations.
```

File: modules/salt/pages/salt-states.adoc [hunk 2]
```diff
@@ -1,97 +1,93 @@
...
+.Example: SLS File Directory Structure
 
 ----
 ├── manager_org_DEVEL
 │   ├── files
 │   │    ... files needed by states (uploaded by users)...
 │   └── state.sls
-         ... other sls files (created by users)...
+         ... other SLS files (created by users)...
 E.g.:
```
Comment (keichwa) [inline modules/salt/pages/salt-states.adoc]:
```suggestion
For example:
```

File: modules/salt/pages/salt-file-locations.adoc [hunk 3]
```diff
@@ -31,79 +31,50 @@ master_tops:
...
 ====
-+
 
-/srv/susemanager/salt::
-This directory is created by {productname} and contains assigned channels and packages for clients, groups, and organizations.
-These files will be overwritten and regenerated.
-A good analogy for this directory would be the SUSE Manager database translated into Salt directives.
-+
+[path]``/srv/susemanager/salt``::
+This directory is generated by {productname} and based on assigned channels and packages for clients, groups and organizations.
```
Comment (keichwa) [inline modules/salt/pages/salt-file-locations.adoc]: I think here will be more than 1 file.

**Pull request merged**